reana_server/config.py

```diff
@@ -116,6 +116,10 @@ def _(x):
 APP_DEFAULT_SECURE_HEADERS["content_security_policy"] = {}
 APP_HEALTH_BLUEPRINT_ENABLED = False
 
+# Rate limiting configuration
+# ===========================
+RATELIMIT_ENABLED = False
```

Comment:
Currently hardcoding this value.. but actually, it would be interesting to make it configurable in case someone wants to adjust it. Some thoughts about it:

Limiting with [Invenio-App/Flask-limiter](https://invenio-app.readthedocs.io/en/latest/configuration.html#invenio_app.config.RATELIMIT_APPLICATION):
- _Flexibility_: Configuring means touching many different variables (limiting per endpoint, limiting auth VS anonymous users) which allow fine-tuning.
- _To be adapted to Kubernetes_: [By default the limiter relies on _in memory_ entries to store for the number of requests](https://invenio-app.readthedocs.io/en/latest/configuration.html#invenio_app.config.RATELIMIT_STORAGE_URL) (this would become a problem if RS is scaled up). The solution is easy and it would be to connect the rate limiter to Redis (to be tested).

Limiting with [Traefik rate limiter](https://doc.traefik.io/traefik/middlewares/ratelimit/):
- _Less-flexibility_: We would have less granularity as doing it inside the application.
- _Closer to prod setup_: It would be more similar to what we do in prod with HAProxy.
- _More harmony_: the approach of rate-limiting inside the app wouldn't work for Interactive sessions (and future entities like them), Traefik would.